### PR TITLE
03 bash fix owner

### PR DESCRIPTION
**Pull request by DomninDmitry-dev**
Search for files / folders with id and group-id in the user's home directory that do not match the id and group-id of this user.

### CODE REVIEW

File: 03-Bash/fix_owner/fix_owner.sh [hunk 1]
```diff
@@ -7,12 +7,16 @@ if [ -n "$1" ]; then
 			;;
 		-h | --help)
 			echo "Usage: fix_owner.sh [OPTION]"
-			echo -e "...\n"
+			echo -e "Search for files / folders with ID and GROUP_ID in the user's home directory that do not match the FILE_ID and FILE_GROUP_ID of this user.\n"
```
Comment (yekovalyov) [inline 03-Bash/fix_owner/fix_owner.sh]: Too long line

**Comment by yekovalyov** [comment]
Please check your commits using checkpatch.pl from Linux repository.

**Comment by DomninDmitry-dev** [comment]
Ok, now I will check

**Comment by DomninDmitry-dev** [comment]
Done

**Pull request merged**